specification/billing/resource-manager/Microsoft.Billing/preview/2020-11-01-preview/billingPromotions.json

```diff
@@ -313,7 +349,37 @@
         }
       }
     },
+    "PromotionsCheckEligibilityResponse": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PromotionCheckEligibilityResponsePropertiesList"
+        }
+      }
+    },
     "PromotionCheckEligibilityResponseProperties": {
+      "type": "object",
+      "properties": {
+        "promotion": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PromotionEligibilityProperties"
+        }
+      }
+    },
+    "PromotionCheckEligibilityResponsePropertiesList": {
+      "type": "object",
+      "properties": {
+        "promotions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PromotionEligibilityProperties"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "PromotionEligibilityProperties": {
       "type": "object",
       "properties": {
         "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

What does id of the Promotion represent? Is it a reference of PromotionResponse ARM Resource? If yes, please use: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details).

Reply:
I dont think this is a ARM resource... @rkapso can you please confirm?